sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/mSers/MSersDirectorySteps.java

```diff
@@ -149,5 +155,39 @@ public MSersDirectorySteps(
               "There are less results than expected");
           softly.assertAll();
         });
+
+    And(
+        "^I filter reports by \"([^\"]*)\" region and \"([^\"]*)\" district$",
+        (String region, String district) -> {
+          webDriverHelpers.waitUntilIdentifiedElementIsPresent(REGION_FILTER_COMBOBOX);
+          webDriverHelpers.selectFromCombobox(REGION_FILTER_COMBOBOX, region);
+          webDriverHelpers.selectFromCombobox(DISTRICT_FILTER_COMBOBOX, district);
+          webDriverHelpers.clickOnWebElementBySelector(AGGREGATED_REPORT_APPLY_FILTER_BUTTON);
+        });
+
+    And(
+        "^I click on edit first aggregated report from grid from mSERS$",
+        () -> {
+          webDriverHelpers.waitForPageLoadingSpinnerToDisappear(30);
+          webDriverHelpers.waitUntilIdentifiedElementIsVisibleAndClickable(
+              FIRST_AGGREGATED_REPORT_EDIT_BUTTON);
+          webDriverHelpers.clickOnWebElementBySelector(FIRST_AGGREGATED_REPORT_EDIT_BUTTON);
+          webDriverHelpers.waitUntilIdentifiedElementIsPresent(EDIT_AGGREGATED_REPORT_HEADER);
+        });
+
+    And(
+        "^I check that first row contains \"([^\"]*)\" in column (\\d+)$",
+        (String expectedResult, Integer columnNumber) -> {
+          String actualResult =
+              webDriverHelpers.getTextFromWebElement(
```

Comment:
why new locator here?

Reply:
getElementFromFirstRowAndSpecificColumnByColumnNumber created and moved to MSersDirectoryPage.java